op-service/testutils/mock_l1.go

```diff
@@ -37,3 +37,34 @@ func (m *MockL1Source) L1BlockRefByHash(ctx context.Context, hash common.Hash) (
 func (m *MockL1Source) ExpectL1BlockRefByHash(hash common.Hash, ref eth.L1BlockRef, err error) {
 	m.Mock.On("L1BlockRefByHash", hash).Once().Return(ref, err)
 }
+
+type MockL1Client struct {
+	MockEthClient
+}
+
+func (m *MockL1Client) L1BlockRefByLabel(ctx context.Context, label eth.BlockLabel) (eth.L1BlockRef, error) {
+	out := m.Mock.MethodCalled("L1BlockRefByLabel", label)
+	return out[0].(eth.L1BlockRef), *out[1].(*error)
+}
```

Comment:
This is similar to `MockL2Client` which we already have.

I am looking at `MockL1Source` and trying to understand why it is different, and if we can unify `MockL1Source` and `MockL1Client`.